### DISCUSSION TITLE
Unable to load folder in network path

### DISCUSSION THREAD

**Discussion by kevinxxng** [phcode-dev/phoenix#1756] [General]
i have project files in other server where I access it with network path like \\10.x.x.x\project1 . The same folder can be open in Brackets but not in Phoenix Code , is this a bug?

**Comment by abose** [comment]
We are yet to support network drives directly in Windows. See tracking issue: https://github.com/phcode-dev/phoenix/issues/1757
This will most likely be fixed in the next release.

A workaround is to assign the network drive to an actual drive letter with
1. https://kb.netgear.com/19864/How-do-I-map-a-network-drive-in-Windows
2. https://support.microsoft.com/en-us/windows/map-a-network-drive-in-windows-29ce55d1-34e3-a7e2-4801-131475f9557d

Please comment if the problem is not in windows OS as described above.

**Comment by brazenCoding** [comment]
October 15th, still the same behavior. Will try to the drive letter work-around.
